pkg/templates/externalccm/hetzner.go

```diff
@@ -183,6 +193,11 @@ func hetznerDeployment() *appsv1.Deployment {
 										},
 									},
 								},
+								{
+									Name: "HCLOUD_NETWORK",
+									// TODO(xmudrii): Should we keep the network ID/name in the secret?
+									Value: networkID,
```

Comment:
Should this be in a secret or this approach is okay?

Reply:
I don't know why networkID should be treated as a secret. So it's OK like this.